service/src/main/kotlin/fi/espoo/evaka/invoicing/domain/Invoices.kt

```diff
@@ -101,6 +101,7 @@ data class InvoiceRowDetailed(
     val periodEnd: LocalDate,
     val product: ProductKey,
     val unitId: DaycareId,
+    val daycareType: String,
```

Comment:
In the database this is an array of `care_types`, and I think it should be reflected in the Kotlin side, too. That is, the type here would be `List<CareType>`.